packages/core/src/components/forms/_input.scss

```diff
@@ -29,6 +29,35 @@ Styleguide input
     @include pt-input-large();
   }
 
+  &.#{$ns}-small {
+    .#{$ns}-button {
```

Comment:
❌ `<input>` tags do not support any children so these child selectors are not necessary.